### ISSUE TITLE
Can't find Wine in menu

### ISSUE THREAD

**Issue by nikoloiz** [Botspot/pi-apps#1687]
### Confirmations

- [X] I searched for similar bug reports in https://github.com/Botspot/pi-apps/issues and found none were relevant.
- [ ] I tried updating Pi Apps and the issue is still not fixed. [How to update Pi Apps?](https://github.com/Botspot/pi-apps#to-update-pi-apps)

### What happened?

I am trying to install Wine and it isn't in the app list.

### Description

I was trying to install Wine on my Raspberry Pi 400 running Twister OS and cannot find them in the menus and it does not show up when i search for it. I tried running the bash script at ~/pi-apps/apps/'Wine (x86)'/install-32.sh but it doesn't install.

### What is your operating system?

Twister OS 2.1

### (Optional) Error log? Terminal output? Debug messages?

```bash
No memory split information due to running a 64-bit kernel. Continuing...
./install-32: line 134: /manage: No such file or directory
./install-32: line 134: error: command not found
Installed Box86 version:
Box86 with Dynarec v0.2.1 aa7b620d built on Apr  7 2021 17:38:04
./install-32: line 148: install_packages: command not found
```

**Comment by Botspot** [comment]
Pi-Apps detects that you are running TwisterOS and hides Wine to prevent compatibility problems.

TL;DR you should already have Wine installed.

**Comment by nikoloiz** [comment]
Thanks for the help!
